lab_3_keywords_textrank/main.py

```diff
@@ -385,7 +455,11 @@ def add_edge(self, vertex1: int, vertex2: int) -> int:
                 0 if edge was added successfully, otherwise -1
         In case of vertex1 being equal to vertex2, -1 is returned as loops are prohibited
         """
-        pass
+        if vertex1 == vertex2:
+            return -1
+        self._edges[vertex1] = self._edges.get(vertex1, []) + [vertex2]
+        self._edges[vertex2] = self._edges.get(vertex2, []) + [vertex1]
+        return 0
 
     # Step 7.2
     def is_incidental(self, vertex1: int, vertex2: int) -> int:
```

Comment:
in the task description it is specified, that  _if either of vertices is not present in the graph, -1 is returned_

Reply:
done